docs/guide/02-writing-templates.md

```diff
@@ -267,26 +274,47 @@ Which repeat is more efficient depends on your use case: if updating the DOM nod
 
 ### Rendering nothing
 Sometimes, you may want to render nothing at all. The values `undefined`, `null` and the empty string (`''`) in a text binding all render an empty text node. In most cases, that's exactly what you want:
-````js
+
+```js
 import {html} from 'lit-html';
 ${user.isAdmin
       ? html`<button>DELETE</button>`
       : ''
   }
-````
-The DOM is inconsistent in some cases with its behavior towards `undefined` & `null`. Using an empty string (`''`) is the most consistent. 
+```
+
+In some cases, you want to render nothing at all. In these cases, you can use the `nothing` value provided by lit-html.
+
+```js
+import {html, nothing} from 'lit-html';
+${user.isAdmin
+      ? html`<button>DELETE</button>`
+      : nothing
+  }
+```
+
+In this case, when `user.isAdmin` is false, no text node is rendered.
+
 #### nothing and the slot fallback content
-`nothing` is a sentinel value provided by lit-html. It really does render nothing, because
-it clears the Part.
 
-Why is this important? Imagine you have a shadow DOM enabled custom element, `shadow-element`, that uses a slot. 
-The template looks like this:
+One specific use case where an empty text node causes issues is when you're using a `<slot>` element inside a shadow root. 
+
+<div class="alert alert-info">
+
+**Not using shadow DOM?** Understanding this use case requires some knoweldge of shadow DOM and slots. On the bright side, if you're not using lit-html to render into shadow DOM, you will probably never need to use `nothing`, and you can skip this section.
```

Comment:
Is this note needed? I feel like this could be a note about any feature... not using it? Don't worry about it! :)

Maybe instead of a note, include links to Shadow DOM docs on MDN like https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM

Reply:
I think the key point here is that this is in the middle of the lit-html docs, which mostly don't talk about shadow DOM at all; this one section gets deep into the weeds of shadow DOM and slots, which confused the original reporter.

(If this was in the LitElement docs, I'd agree, since LitEl uses shadow DOM by default, and we talk about it in many places.)

Reply:
Came back to this after a couple of days, I think I agree with you. I also didn't really like the tone. So I'm rewriting this a bit.